src/controlCenter/widgets/backlight/backlightUtil.vala

```diff
@@ -0,0 +1,113 @@
+namespace SwayNotificationCenter.Widgets {
+    class BacklightUtil {
+
+        [DBus (name = "org.freedesktop.login1.Session")]
+        interface Login1 : Object {
+            public abstract void set_brightness (string subsystem, string name, uint32 brightness) throws GLib.Error;
+        }
+
+        string path_current;
+        string path_max;
+        File fd;
+        FileMonitor monitor;
```

Comment:
Should be null as default and should be checked if null due to the chance that the try catch statement below might fail